Neos.Fusion/Classes/Core/Runtime.php

```diff
@@ -227,6 +270,15 @@ public function getCurrentContext()
         return $this->currentContext;
     }
 
+    /**
+     * @param string $name the key of the variable like `request` or `node`.
+     * @return mixed null if not set
+     */
+    public function getContextVariable(string $name): mixed
+    {
+        return $this->currentContext[$name] ?? $this->defaultContextVariables->value[$name] ?? null;
+    }
```

Comment:
Alternatively (or rather additionally) suggested: 

```php
public function getContextObjectOfType(string $name, string $classStringName): object
{
    $value = $this->getContextVariable($name, $classStringName);
    if (!$value instanceof ActionRequest) {
        throw new \RuntimeException(sprintf('Expected Fusion variable "request" to be of type ActionRequest, got value of type "%s".', $classStringName, get_debug_type($value)), 1693557504154);
    }
    return $value;

    return $this->currentContext[$name] ?? $this->defaultContextVariables->value[$name] ?? null;
}
```

Reply:
Will be discussed here https://github.com/neos/neos-development-collection/issues/4475